.github/workflows/action_server_tests.yml

```diff
@@ -51,8 +51,8 @@ jobs:
       with:
         python-version: ${{ matrix.python }}
         cache: poetry
-    - name: Install invoke/tomlkit
-      run: pip install invoke tomlkit
+    - name: Install devutils requirements
+      run: python -m pip install -r ../devutils/requirements.txt
```

Comment:
So the default current working directory is actually _.github_ while `CURDIR` points to _.github/workflows_, right?

Maybe we can also use a `ROOT = CURDIR.parent.parent` in case we use it often. (and `CURDIR` can refactor to `WORKFLOWS_DIR` as it is a little bit misleading when using it in the code and can trick you)

But your call here, if it works fine as it is, no must for a change.